tests/phpunit/tests/meta/registerMeta.php

```diff
@@ -512,6 +582,518 @@ public function filter_get_object_subtype_for_customtype( $subtype, $object_id )
 		return 'even';
 	}
 
+	public function data_get_default_data() {
+		return array(
+			array(
+				array(
+					'single'  => true,
+					'default' => 'wibble',
+				),
+				true,
+				'wibble',
+			),
+			array(
+				array(
+					'single'  => true,
+					'default' => 'wibble',
+				),
+				false,
+				array( 'wibble' ),
+			),
+			array(
+				array(
+					'single'  => false,
+					'default' => 'wibble',
+				),
+				true,
+				'wibble',
+			),
+			array(
+				array(
+					'single'  => false,
+					'default' => 'wibble',
+				),
+				false,
+				array( 'wibble' ),
+			),
+			array(
```

Comment:
This doesn't seem right. If I'm registering a single meta key that is an array. If I then ask for it as a multiple meta key, the value should be `array( array( 'wibble' ) )`.